wmfdata/hive.py

```diff
@@ -170,34 +149,40 @@ def load_csv(
     """
 
     load_table_cmd = """
-    LOAD DATA LOCAL INPATH "{path}"
+    LOAD DATA INPATH "hdfs://{path}"
     OVERWRITE INTO TABLE {db_name}.{table_name}
     """
 
     try:
+        __, tmp_path = tempfile.mkstemp()
         if headers:
-            __, tmp_path = tempfile.mkstemp()
             with open(path, 'r') as source, open(tmp_path, 'w') as target:
                 # Consume the first line so it doesn't make it to the copy
                 source.readline()
                 copyfileobj(source, target)
             path = tmp_path
 
+        # Copy the file to HDFS because pyhive runs via JDBC not off the local client
+        hdfs_path = f"/tmp/{path}"
```

Comment:
There are a couple issues here.

1. `msktemp` returns a path like `/tmp/tmpdw4b3ubr`. That would give us `/tmp//tmp/tmpdw4b3ubr` as our `hdfs_path`.
2. When I tried running this, I got an error because `hdfs:///tmp/tmp` is owned by milimetric, probably because it was created with default permissions when testing this code. If we put the file directly in `/tmp` (we can even use the `mkstemp` path without modification for HDFS), that should avoid the problem.

Reply:
good point, I just removed the '/tmp' prefix I think that fixes it.

Reply:
@nshahquinn I tested this on my notebooks and it seems like it's working ok.  Let me know if there's anything else.